src/xcode/ENA/ENA/Source/Scenes/AppInformation/AppInformationViewController+DynamicTableViewModel.swift

```diff
@@ -48,101 +48,111 @@ private extension DynamicAction {
+						   height: 230),
 			cells: [
-				.title2(text: "App_Information_About_Title".localized),
-				.headline(text: "App_Information_About_Description".localized),
-				.subheadline(text: "App_Information_About_Text".localized)
+				.title2(text:  AppStrings.AppInformation.aboutTitle),
+				.headline(text: AppStrings.AppInformation.aboutDescription),
+				.subheadline(text: AppStrings.AppInformation.aboutText)
 			]
 		)
 	])
 
 	private static let contactModel = DynamicTableViewModel([
 		.section(
-			header: .image(UIImage(named: "Illu_Appinfo_Kontakt"), height: 230),
+			header: .image(UIImage(named: "Illu_Appinfo_Kontakt"),
+						   accessibilityLabel: AppStrings.AppInformation.contactImageDescription,
+						   height: 230),
 			cells: [
-				.title2(text: "App_Information_Contact_Title".localized),
-				.body(text: "App_Information_Contact_Description".localized),
-				.headline(text: "App_Information_Contact_Hotline_Title".localized),
-				.phone(text: "App_Information_Contact_Hotline_Text".localized, number: "App_Information_Contact_Hotline_Number".localized),
-				.footnote(text: "App_Information_Contact_Hotline_Description".localized),
-				.footnote(text: "App_Information_Contact_Hotline_Terms".localized)
+				.title2(text: AppStrings.AppInformation.contactTitle),
+				.body(text: AppStrings.AppInformation.contactDescription),
+				.headline(text: AppStrings.AppInformation.contactHotlineTitle),
+				.phone(text: AppStrings.AppInformation.contactHotlineText, number: AppStrings.AppInformation.contactHotlineNumber),
+				.footnote(text: AppStrings.AppInformation.contactHotlineDescription),
+				.footnote(text: AppStrings.AppInformation.contactHotlineTerms)
 			]
 		)
 	])
 
 	private static let imprintModel = DynamicTableViewModel([
 		.section(
-			header: .image(UIImage(named: "Illu_Appinfo_Impressum"), height: 230),
+			header: .image(UIImage(named: "Illu_Appinfo_Impressum"),
+						   accessibilityLabel: AppStrings.AppInformation.imprintImageDescription,
+						   height: 230),
 			cells: [
-				.headline(text: "App_Information_Imprint_Section1_Title".localized),
-				.body(text: "App_Information_Imprint_Section1_Text".localized),
-				.headline(text: "App_Information_Imprint_Section2_Title".localized),
-				.body(text: "App_Information_Imprint_Section2_Text".localized),
-				// .headline(text: "App_Information_Legal_Section3_Title".localized),
-				.body(text: "App_Information_Imprint_Section3_Text".localized),
-				.headline(text: "App_Information_Imprint_Section4_Title".localized),
-				.body(text: "App_Information_Imprint_Section4_Text".localized)
+				.headline(text: AppStrings.AppInformation.imprintSection1Title),
+				.body(text: AppStrings.AppInformation.imprintSection1Text),
+				.headline(text: AppStrings.AppInformation.imprintSection2Title),
+				.body(text: AppStrings.AppInformation.imprintSection2Text),
+				// .headline(text: AppStrings.AppInformation.legalSection3Title),
```

Comment:
What will collide with #258. Be careful when merging.

Reply:
Updated, will wait to commit until #258 is merged

Reply:
Should have been merged.